### DISCUSSION TITLE
0300. 最长递增子序列

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#93] [Comments]
# 0300. 最长递增子序列 | 算法通关手册

[https://algo.itcharge.cn/solutions/0300-0399/longest-increasing-subsequence/](https://algo.itcharge.cn/solutions/0300-0399/longest-increasing-subsequence/)

**Comment by FuyiHuang** [comment]
如果nums[j]<=nums[i], 则num[i]不可以接在nums[j]后面，这里应该是如果nums[j]>=nums[i]
